node/src/cli.rs

```diff
@@ -110,6 +126,14 @@ pub struct RunCmd {
 	/// Public identity for participating in staking and receiving rewards
 	#[structopt(long, parse(try_from_str = parse_h160))]
 	pub author_id: Option<H160>,
+
+	/// Number of accounts to be funded in the genesis in dev mode (default 10)
+	#[structopt(long, default_value = "10")]
+	pub accounts: u32,
+
+	/// Mnemonic from which we can derive funded accounts in the genesis
+	#[structopt(long)]
+	pub mnemonic: Option<String>,
```

Comment:
These also shouldn't go on the run command.

I think the idea we discussed today is that when you run with `--dev` it will imply the default mnemonic and a fixed number of prefunded accounts. If you don't so `--dev` then you are providing a chain spec which will already have accounts funded at genesis.

Reply:
I might be confused here, but how do I pass a mnemonic without adding it as a flag in the run command?

Reply:
I'm suggesting you add the flags to the `build-spec` command. Then once you've built the spec with the prefunded accounts you want, you can run it by pointing to that spec.

```bash
# First generate a spec
moonbeam build-spec --mnemonic "river wife ... drive" --accounts 15 > my-prefunded-spec.json

# Now run it
moonbeam --chain my-prefunded-spec.json --collator
```

Reply:
got it thanks!